base_site/index.html

```diff
@@ -1,96 +1,222 @@
+  <title>Quantinuum Technical Documentation</title>
+  <meta content="width=device-width, initial-scale=1" name="viewport">
+  <meta content="Webflow" name="generator">
+  <link href="css/normalize.css" rel="stylesheet" type="text/css">
+  <link href="css/webflow.css" rel="stylesheet" type="text/css">
+  <link href="css/docs-6c2398.webflow.css" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com" rel="preconnect">
+  <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin="anonymous">
+  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js" type="text/javascript"></script>
+  <script type="text/javascript">WebFont.load({  google: {    families: ["Inter:regular,500,600,700"]  }});</script>
+  <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
+  <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
+  <link href="images/webclip.png" rel="apple-touch-icon">
 </head>
 <body>
-<header class="bd-header navbar navbar-expand-lg">
-  <div class="bd-header__inner bd-page-width">
-    <div class="col-lg-3 navbar-header-items__start">
-      <div class="navbar-item">
-        <a class="navbar-brand logo" href="#">
-          <img src="_static/Quantinuum_logo_black.png" class="logo__image only-light" alt="Quantinuum Documentation - Home">
-        </a>
-      </div>
-    </div>
-    <div class="col-lg-9 navbar-header-items">
-      <div class="navbar-item">
-        <nav class="navbar-nav">
-          <ul class="bd-navbar-elements navbar-nav">
-            <li class="nav-item">
-              <a class="nav-link nav-internal" href="inquanto/index.html"> InQuanto </a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link nav-internal" href="h-series/index.html"> H-Series </a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link nav-internal" href="tket/index.html"> TKET </a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link nav-internal" href="nexus/index.html"> Nexus </a>
-            </li>
-        </nav>
+  <div class="page-wrapper">
+    <div data-animation="default" data-collapse="medium" data-duration="400" data-easing="ease" data-easing2="ease" role="banner" class="navbar w-nav">
+      <div class="padding-global">
+        <div class="container is-nav">
+          <div class="navbar_logo-wrap">
+            <a href="#" class="navbar_logo w-nav-brand"><img src="images/quantinuumlogo.png" loading="lazy" alt="" class="image-fit-contain"></a>
+            <div>
+              <div>|</div>
+            </div>
+            <div>
+              <div class="text-size-xsmall text-color-gray">Quantinuum Technical Documentation</div>
```

Comment:
This one was fine to leave as DOCS, I think - just the `<title>` element needed expanding.

```suggestion
              <div class="text-size-xsmall text-color-gray">DOCS</div>
```